examples/hyperparam/MLproject

```diff
@@ -6,7 +6,7 @@ entry_points:
   # train Keras DL model
   train:
     parameters:
-      training_data: {type: string, default: "../sklearn_elasticnet_wine/wine-quality.csv"}
+      training_data: {type: string, default: ""}
```

Comment:
I think an empty string default value here may be somewhat confusing. Can we add a small comment indicating that data will be fetched from the web by default?

In an ideal world, I think it'd be nice to support `http`/`https` URLs with the `path` project parameter type. Then, we could rewrite this as:
```
training_data: {type: path, default: "http://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-white.csv"}
```

Perhaps this would make a good first issue.